src/model.py

```diff
@@ -238,6 +263,22 @@ async def generate(self, request):
                 stream = stream.as_numpy()[0]
             else:
                 stream = False
+            exclude_input_in_output = pb_utils.get_input_tensor_by_name(
+                request, "exclude_input_in_output"
+            )
+            if exclude_input_in_output:
+                exclude_input_in_output = exclude_input_in_output.as_numpy()[0]
+            elif exclude_input_in_output is None and stream:
+                exclude_input_in_output = True
```

Comment:
just so I understand, default in stream case is True, default in non stream case is False ?

Reply:
yes, that is correct